modules/client-configuration/pages/autoinst-distributions.adoc

```diff
@@ -106,3 +106,15 @@ You can autoinstall {productname} Proxies in the same way as {sle} clients.
 Make sure you use the {sle} installation media, and choose the [guimenu]``SLE-Product-SUSE-Manager-Proxy-4.2-Pool for x86_64`` base channel.
 ====
 endif::[]
+
+== Handling Kernel Options for Distributions and Profiles
+
+{productname} is able to combine the kernel options that you assign. This is done using a special inheritance logic. There are three object types that are relevant for this:
+
+. Distributions (or short "Distros")
+. Profiles
+. Systems
+
+As a fourth special point of influence on the final kernel options there is also the Cobbler settings file ([path]``/etc/cobbler/settings.yaml``) that defines default kernel options for all distributions.
+
+If you start prefix a key with "!" then the key will be anilathed from the final kernel command line.
```

Comment:
```suggestion
If you prefix an option with [literal]``!`` then the option will be removed from the final kernel command line.
```
I'm not sure whether you actually mean one or more single kernel options here.
Instead of "removed" we maybe could also say "dropped" or "pruned".  Or something similar.